src/Controls/tests/DeviceTests/Elements/Shell/ShellTabBarTests.iOS.cs

```diff
@@ -20,6 +20,11 @@ UITabBar GetTabBar(ShellSection item)
 			var pagerParent = (shell.CurrentPage.Handler as IPlatformViewHandler)
 				.PlatformView.FindParent(x => x.NextResponder is UITabBarController);
 
+			if (OperatingSystem.IsMacCatalystVersionAtLeast(15, 0) || OperatingSystem.IsMacOSVersionAtLeast(15, 0))
```

Comment:
Could include a small comment here? Explain why checking for the operating system version is necessary (In in macOS 15 Sequoia the UITabBar is nested within subviews).

Reply:
@jsuarezruiz, Updated the comment and commited the changes, kindly review and share concerns.